dataland-frontend/src/components/pages/SearchTaxonomy.vue

```diff
@@ -13,12 +16,19 @@ import TheHeader from "@/components/structure/TheHeader"
 import TheContent from "@/components/structure/TheContent"
 import {useRoute} from "vue-router"
 import apiSpecs from "../../../build/clients/backend/backendOpenApi.json";
+import AuthenticationWrapper from "@/components/wrapper/AuthenticationWrapper";
 
 
 export default {
   name: "SearchTaxonomy",
-  components: {SearchTaxonomyHeader, EuTaxoSearchBar, TheHeader, TheContent},
-  data(){
+  components: {
+    SearchTaxonomyHeader,
+    EuTaxoSearchBar,
+    TheHeader,
+    TheContent,
+    AuthenticationWrapper
+  },
+  data() {
     return {
       showIndexPanel: true,
```

Comment:
Diese Variable wird in der DOM gar nicht benutzt.  Wieso gibt es sie dann eigentlich?
Sie wird auch in keiner anderen Methode verwendet, sondern immer nur gesetzt.

Reply:
Bei mir hat lokal alles weiterhin normal ausgesehen, nachdem ich es rausgenommen habe. Daher commite ich die Änderung.